src/modules/storage.js

```diff
@@ -23,22 +27,26 @@ const addList = () => {
       <div class="single-list div-style">
       <form class="single-list-form">
         <input type="checkbox" class="checkbox">
-        <input type="text" id="single-list-items" class="single-list-input main-inputs" value="${list}">
+        <input type="text" class="single-list-input main-inputs" value="${list}">
       </form>
       <div class="single-list-action-button">
-      <button class = "move-btn"><i class="fa-solid fa-ellipsis-vertical"></i></button>
+        <button class = "delete-btn" onclick="removeList('${list}')"><i class="fa-solid fa-trash-can"></i></button>
       </div>
     </div>
             `;
     }
+    // console.log(index);
   });
   listsList.innerHTML = listCode;
   localStorage.setItem('listData', JSON.stringify(listArray));
 };
 
-window.removeBook = (list) => {
+/* eslint-disable no-unused-vars */
+
+window.removeList = (list) => {
   listArray = listArray.filter((elem) => elem.list !== list);
   addList();
+  // console.log('removed');
 };
```

Comment:
- [ ] The remove functionality could not be properly tested because items were not getting added,but please kindly make sure the remove functionality is working and the correct item is getting removed
- [ ] Please also make sure that deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.